src/Components/Preview.lua

```diff
@@ -230,12 +261,31 @@ function Preview:render()
 		}, {
 			Button = e(FloatingButton, {
 				Activated = self.expandSelection,
+				ToolTipText = `Render story in {if self.state.expand then "Hoarcekat" else "Studio viewport"}`,
 				Image = "rbxasset://textures/ui/VR/toggle2D.png",
 				ImageSize = UDim.new(0, 24),
 				Size = UDim.new(0, 40),
 			}),
 		}),
 
+		ClearOutputButton = selectedStory and e("Frame", {
+			AnchorPoint = Vector2.new(1, 1),
+			BackgroundTransparency = 1,
+			Position = UDim2.new(0.99, -90, 0.99),
+			Size = UDim2.fromOffset(40, 40),
+			ZIndex = 2,
+		}, {
+			Button = e(FloatingButton, {

```

Comment:
This needs some more obvious visual tell (such as a darker color) for enabled/disabled.

Reply:
![image](https://github.com/Kampfkarren/hoarcekat/assets/75750304/4c4eefa3-5055-4ed2-87e3-a528f675ab7a)
Looking better